src/components/Article.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import { getArticles, getArticlesById } from "../api";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import { Comments } from "./Comments";
+
+export const Article = () => {
+  const params = useParams();
+  const [article, setArticle] = useState(null);
+  const [loadingArticle, setLoadingArticle] = useState(false);
+
+  useEffect(() => {
+    setLoadingArticle(true);
+    getArticlesById(params.article_id).then((data) => {
+      setArticle(data.articles);
+      setLoadingArticle(false);
+    });
+  }, [params.article_id]);
+
+  if (!article) {
+    return null;
+  }
+
+  return (
+    <article>
+      {loadingArticle ? (
+        "Article is loading..."
+      ) : (
+        <div className="article-page">
```

Comment:
Try and avoid using divs where possible. See if you can think of a more semantic html tag :)

Reply:
changed some of the divs to p tags.